parsl/tests/manual_tests/test_scaling.py

```diff
@@ -0,0 +1,82 @@
+import parsl
```

Comment:
Run this test in CI - it takes 25s on my laptop which I think is time worth spending.

Reply:
I used this test when I was debugging. This test passed when there were multiple bugs. 

For instance:
1. This test checked the min_blocks at the end of test, but that is actually resulted from case 1 in strategy (when there is no task), not case 4
2. This test does not check if the strategy follows `max_idletime` to kill blocks (from case 4 in strategy).

I had another test in this PR that is a bit more comprehensive, but that one takes ~50s. I could try to change the time to make it shorter.

Reply:
I think 50s is worth spending too.